cvat-ui/src/components/create-task-page/advanced-configuration-form.tsx

```diff
@@ -178,6 +188,33 @@ class AdvancedConfigurationForm extends React.PureComponent<Props> {
         );
     }
 
+    private renderSortingMethodRadio(): JSX.Element {
+        return (
+            <Form.Item
+                label='Sorting method'
+                name='sortingMethod'
+                rules={[
+                    {
+                        required: true,
+                        message: 'The field is required.',
+                    },
+                ]}
+                help='Specify how to sort images. It is not relevant for videos.'
+            >
+                <Radio.Group>
+                    <Radio value={SortingMethod.LEXICOGRAPHICAL} key={SortingMethod.LEXICOGRAPHICAL}>
+                        Lexicographical
+                    </Radio>
+                    <Radio value={SortingMethod.NATURAL} key={SortingMethod.NATURAL}>Natural</Radio>
+                    <Radio value={SortingMethod.PREDEFINED} key={SortingMethod.PREDEFINED}>
+                        Predefined
+                    </Radio>
```

Comment:
Maybe to add some hints would be a good idea. It might be not really understandable for a user what are NATURAL and PREDEFINED sortings.

Reply:
I've added info with examples into documentation. Should I also display this help information in ui? Natural sorting is official name. [wiki](https://en.wikipedia.org/wiki/Natural_sort_order)

Reply:
If Natural is official name, let's keep as is.